### ISSUE TITLE
Feathers Socket work normal on localhost but not in Google Cloud Host

### ISSUE THREAD

**Issue by radiegtya** [feathersjs/feathers#800]
My Feathers Socket work normal on localhost but not in Google Cloud Host, it said:

"Possible Unhandled Promise Rejection, Socket timeout"...

Please help

**Comment by edwardsmarkf** [comment]
what does your socket.io command look like?   you might try just ':3030'
instead?

note - i am also using google-cloud host.


Thank you,

Mark Edwards

**Comment by radiegtya** [comment]
Here's my socket config

```
import feathers from '@feathersjs/feathers';
import socketio from '@feathersjs/socketio-client';
import io from 'socket.io-client';
import Config from 'react-native-config';

//Note Config API URL is from google: https://myapp-12999.appspot.com
const socket = io(Config.API_URL, { transports: ['websocket'], forceNew: true });

const client = feathers()
  .configure(socketio(socket, {
    pingInterval: 10000,
    pingTimeout: 50000
  }));

export default client;

```

Nothing special, anyway is there something that you setting up on google cloud, such as "enabling socket via yaml config?"

Thank @edwardsmarkf

**Comment by edwardsmarkf** [comment]
hi eja -

for fun, can you please change:

const socket = io(Config.API_URL, { transports: ['websocket'],
forceNew: true });

to

const socket = io(':3030', { transports: ['websocket'], forceNew: true });

and see what happens?




Thank you,

Mark Edwards

**Comment by radiegtya** [comment]
@edwardsmarkf 

ofc it will not working, it needs the API URL from google, while your string is just a port.. 
Afterall my google doesn't use port 3030

**Comment by daffl** [comment]
Thanks for helping @edwardsmarkf. Judging from https://medium.com/google-cloud/highly-available-websockets-on-google-cloud-c74b35ee20bc and https://stackoverflow.com/questions/37401250/how-do-i-deploy-socket-io-to-google-app-engine there is quite a bit of configuration necessary to enabled websockets in general.

Unfortunately I am not sure if this is the best place to get help for it since it isn't really a Feathers issue. If you are on a paid plan I would try and contact Google to see if they can help.

**Comment by bgold0** [comment]
@radiegtya did you figure this out?
